### PR TITLE
New canvas onboarding

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
QA:
- Load the canvas and see the new canvas onboarding
- Change the screen size and check it works
- The plus does not click yet as there is no header

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2842/
Test PASSed.

File: jujugui/static/gui/src/app/views/topology/viewport.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ YUI.add('juju-topology-viewport', function(Y) {
       // smallest size we accept--no smaller or bigger--or else the
       // presence or absence of scrollbars may affect our calculations
       // incorrectly.  The real canvas size will be set in a moment.
-      canvas.setStyles({height: '600px', width: '800px'});
+      debugger;
```
Comment (makyo) [inline jujugui/static/gui/src/app/views/topology/viewport.js]: Remove

File: jujugui/static/gui/src/app/views/topology/viewport.js [hunk 2]
```diff
@@ -129,6 +129,7 @@ YUI.add('juju-topology-viewport', function(Y) {
       // This sets the minimum viewport size - y was reduced to 200 to render
       // properly on 7" tablets in horizontal view.
       var dimensions = utils.getEffectiveViewportSize(true, 800, 200);
+      debugger;
```
Comment (makyo) [inline jujugui/static/gui/src/app/views/topology/viewport.js]: Remove

**Comment by makyo** [comment]
:+1: with console+debuggers removed QA okay

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2846/
Test PASSed.

File: jujugui/static/gui/src/app/assets/css/base.scss [hunk 3]
```diff
@@ -436,34 +435,56 @@ th {
...
+        position: relative;
+        left: -20px;
+        top: 35px;
+        box-sizing: border-box;
+
+        &:after {
+            content: "";
+            background: #fff;
+            width: 10px;
+        	height: 10px;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/base.scss]: Indent?

File: jujugui/static/gui/src/app/views/topology/panzoom.js [hunk 4]
```diff
@@ -260,6 +260,7 @@ YUI.add('juju-topology-panzoom', function(Y) {
       topo.set('translate', Y.mix(evt.translate));
       vis.attr('transform', 'translate(' + topo.get('translate') + ')' +
               ' scale(' + topo.get('scale') + ')');
+      console.log(topo.get('translate'));
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/topology/panzoom.js]: Another log :)

**Comment by huwshimi** [comment]
Nice one! :+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2847/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2848/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2849/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2850/
Test PASSed.

**Comment by anthonydillon** [comment]
:shipit:

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**